README.md

```diff
@@ -271,9 +281,20 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 "state_timeout_in_seconds": 600,
 
                 // The name of the OIDC claim which provides the mapping of an
-                // OIDC user to an iRODS user account
+                // OIDC user to an iRODS user account.
+                // "irods_user_claim" and "user_attribute_mapping" cannot be used together.
                 "irods_user_claim": "irods_username",
 
+                // The mapping of a user to the provided values. All values must
+                // be matched to map an OIDC user to an iRODS user account.
+                // "irods_user_claim" and "user_attribute_mapping" cannot be used together.
+                "user_attribute_mapping": {
+                    "irods_username": {
+                        "sub": "123-abc-456-xyz",
+                        "email": "rods_user@email.example"
```

Comment:
could use example.org as the domain

Reply:
Just to be sure, prefer `example.org` in the docs, right?

Reply:
right, this is the README.md, yes?

Reply:
Changed it!